src/caselawclient/xquery/resolve_from_identifier.xqy

```diff
@@ -0,0 +1,17 @@
+xquery version "1.0-ml";
+
+declare namespace xdmp="http://marklogic.com/xdmp"; 
+declare variable $identifier_uri as xs:string external;
+declare variable $published_only as xs:int? external := 0;
```

Comment:
Is it safer for this to default to being true, so we have to explicitly request that it include unpublished stuff?